src/index.mjs

```diff
@@ -9,7 +9,7 @@
  * @returns {boolean}
  */
 
-export default function(file, acceptedFiles) {
+export function accept(file, acceptedFiles) {
```

Comment:
This is a breaking change and I don't think it's necessary if we're not making any changes but adding esm support.

Reply:
ESM by nature doesn't like default exports and it is best practice to handle exports as named exports vs. default exports

Reply:
Wouldn't this change CJS usage too? Having to write `require('attr-accept').accept` seems annoying. I think a default export would be fine... I'm not sure there's [consensus](https://danlevy.net/should-you-use-named-or-default-exports/).

Reply:
Since default exports are in the spec, we should avoid making a breaking change now.